File: Prime_Number.py

# Finding a prime number
import math

number = int(input())
i = 2
sqrt_number = int(math.sqrt(number))
while i <= sqrt_number:
	if number % i == 0:
		print('{} is Not a Prime number'.format(number))
		break
	i = i + 1 
else:
	print('{} is a Prime number'.format(number))

File: Prime_Number.cpp
/* Finding a prime number*/

#include<iostream>
#include<cmath>
using namespace std;
int main()
{
	int number, i = 2, flag=0, sqrt_number;
	cout<<"Enter the number : ";
	cin>>number;
	sqrt_number = int(sqrt(number));
	while(i <=  sqrt_number)
	{
		if(number % i == 0)
		{
			cout<<number<<" is a Not a Prime number.";
			flag = 1;
			break;
		}
		i = i + 1;
	}
	if(flag == 0)
		cout<<number<<" is a Prime number.";
	cout<<"\n";
	return 0;
}